.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "trailingComma": "all"
```

Comment:
I think it would be great to create public repository with shared prettier configuration so we will ensure that all our projects are formatted in the same way - it's not the scope of this pull request but we should consider it for the future.

references:
https://prettier.io/docs/en/configuration.html#sharing-configurations